.github/scripts/launch-ec2.sh

```diff
@@ -13,8 +13,10 @@ if ! aws sts get-caller-identity &>/dev/null; then
   exit 1
 fi
 
+# Used by aws CLI
+${AWS_REGION:?Environment variable AWS_REGION must be set}
+
 # Set variables
-AWS_REGION="us-east-2"
```

Comment:
Indeed this isnt helpful, but I've added an env var requirement for this instead